twisted/conch/ssh/transport.py

```diff
@@ -1539,6 +1700,8 @@ def _ssh_KEXDH_REPLY(self, packet):
     def ssh_KEX_DH_GEX_GROUP(self, packet):
         """
         This handles different messages which share an integer value.
+       
+        If it is ECDH, call C{_ssh_KEX_ECDH_REPLY} and pass along the packet.
```

Comment:
This is an ostensibly public method, whose docstring is referencing a private implementation detail.  Why is it interesting to the caller to know this?  A maintainer could just read the code and see what it calls.  It doesn't talk about `_ssh_KEXDH_REPLY`, so this is also asymmetrical.